internal/core/src/exec/expression/Expr.h

```diff
@@ -280,6 +280,22 @@ class SegmentExpr : public Expr {
         return result;
     }
 
+    template <typename T, typename FUNC, typename... ValTypes>
+    void
+    ProcessIndexChunksV2(FUNC func, ValTypes... values) {
```

Comment:
not use v2 name,  why not use upper ProcessIndexChunk mode?

Reply:
ProcessIndexChunk will move the cursor, but I only need to iterate all index chunks.